CHANGES.md

```diff
@@ -1,3 +1,10 @@
+# next next -- TBA (this section will go away but this change isn't going in the current next)
```

Comment:
Indeed, let's make this section go away :)

Reply:
It has now (as part of merging with the release)